microsoft/testsuites/nvme/nvme.py

```diff
@@ -383,3 +363,71 @@ def nvme_rescind_validation(self, node: Node) -> None:
         lspci.disable_devices(device_type="NVME")
         # 2. Enable NVME device.
         lspci.enable_devices()
+
+    @TestCaseMetadata(
+        description="""
+        This test case will
```

Comment:
This description includes steps, but missed the test purpose.

Reply:
fixed